front/src/components/boxs/device-in-room/device-features/SensorDeviceFeature.jsx

```diff
@@ -23,7 +24,7 @@ const SensorDeviceType = ({ children, ...props }) => (
     </td>
     <td>{props.deviceFeature.name}</td>
     {SPECIAL_SENSORS.indexOf(props.deviceFeature.category) === -1 && (
-      <td class="text-right">
+      <td class={cx('text-right', { 'text-nowrap': props.deviceFeature.last_value })}>
```

Comment:
Can you do a stricter type check on this ? If last_value = 0, (ex: temperature = 0°C) then it'll be "false" here, but we want true. 

check only if null